tests/bluetooth/iso/bsim/compile.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+# Copyright 2023 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+# Compile all the applications needed by the bsim tests in these subfolders
+
+INCR_BUILD=1
+
+#set -x #uncomment this line for debugging
+set -ue
+: "${ZEPHYR_BASE:?ZEPHYR_BASE must be set to point to the zephyr root directory}"
+
+source ${ZEPHYR_BASE}/tests/bsim/compile.source
+
+app=../../ncs/nrf/tests/bluetooth/iso/bsim compile
```

Comment:
CI will use the branch-name instead of a ncs/ folder as a root for the NCS checkout, so this can't be hardcoded